.github/workflows/windows.yml

```diff
@@ -168,3 +168,106 @@ jobs:
+        with:
+          python-version: '3.12'
+          architecture: 'arm64'
+          
+      - name: Common Setup
+        uses: ./.github/windows_arm64_steps
+
+      - name: pip-packages
+        run: |
+          pip install git+https://github.com/mesonbuild/meson.git@ddc01cf5d9240cdb0edc4b44e837260948441d12
+          pip install numpy cython pybind11 pythran ninja pytest pytest-xdist pytest-timeout pooch spin hypothesis
+          python -m pip install -r requirements/openblas.txt
+
+      - name: Build
+        run: |
+          $env:cc = "clang-cl"
+          $env:cxx = "clang-cl"
+          $env:fc = "flang-new"
+          spin build --with-scipy-openblas
+
+      - name: Test
+        run: |
+          $env:OPENBLAS_NUM_THREADS = 1
+          spin test -j2 -- --durations=25
+          
+
+  #############################################################################
+  full_build_sdist_wheel_arm64:
+    name: no pythran & sdist+wheel, full, py3.11/npPre, pip+pytest (Win-ARM64)
+    runs-on: windows-11-arm
+    needs: get_commit_message
+    # if: >
+    #   needs.get_commit_message.outputs.message == 1 &&
+    #   (github.repository == 'scipy/scipy' || github.repository == '')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+  
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+          architecture: 'arm64'
+          cache: 'pip'
+          cache-dependency-path: 'environment.yml'
+  
+      - name: Common Setup
+        uses: ./.github/windows_arm64_steps
+  
+      - name: Install OpenBLAS
+        shell: bash
+        run: |
+          set -xe
+          export PKG_CONFIG_PATH=$(cygpath -u "$PKG_CONFIG_PATH")
+          export PATH="$(cygpath -u 'C:\vcpkg\installed\arm64-windows\tools\pkgconf'):$PATH"
+          python -m pip install -r requirements/openblas.txt
+          bash tools/wheels/cibw_before_build_win.sh .
+          echo "PKG_CONFIG_PATH=$(cygpath -w "${{ github.workspace }}")" >> $GITHUB_ENV
+  
+      - name: pip-packages
+        run: |
+          pip install git+https://github.com/mesonbuild/meson.git@ddc01cf5d9240cdb0edc4b44e837260948441d12
+          python -m pip install build delvewheel cython pybind11 meson-python ninja pytest pytest-xdist pytest-timeout pooch hypothesis numpy
+  
+      - name: Build
+        shell: bash
+        run: |
+          set -xe
+          export CC=clang-cl
+          export CXX=clang-cl
+          export FC=flang-new
+          export PATH="$(cygpath -u 'C:\vcpkg\installed\arm64-windows\tools\pkgconf'):$PATH"
+          python -m build --no-isolation -x -Csetup-args="-Duse-pythran=false"
+          wheel_name=$(ls dist/*.whl)
+          openblas_dir=$(python -c"import scipy_openblas32 as sop; print(sop.get_lib_dir())")
+          delvewheel repair --add-path "$openblas_dir" --no-dll libsf_error_state.dll -w wheelhouse "$wheel_name"
```

Comment:
`libsf_error_state.dll` is gone, so that part shouldn't need to be there anymore.